### PR TITLE
[doc] rail_section init

### PR DESCRIPTION
**Pull request by dvdn**
This PR adds rail_section format example in swagger documentation

ref: bot-2416

You can read the result
https://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/doc-swagger-rail-section-bot-2416/documentation/swagger.yml

### CODE REVIEW

**Comment by pthegner** [comment]
https://github.com/CanalTP/Chaos/blob/04bf510ea4708e00042f24e787a1a0d7399dea51/documentation/swagger.yml#L2458 and https://github.com/CanalTP/Chaos/blob/04bf510ea4708e00042f24e787a1a0d7399dea51/documentation/swagger.yml#L2807 missing

File: documentation/swagger.yml [hunk 1]
```diff
@@ -2553,6 +2553,95 @@ components:
...
+              type: array
+              items:
+                type: object
+                properties:
+                  id:
+                    type: string
+                  order:
+                    type: string
+                    enum:
+                      - integer
```
Comment (pthegner) [inline documentation/swagger.yml:2609]:
```suggestion
                    type: integer
                    minimum: 0
```

File: documentation/swagger.yml [hunk 2]
```diff
@@ -2553,6 +2553,95 @@ components:
...
+                type: array
+                items:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                    order:
+                      type: string
+                      enum:
+                        - integer
```
Comment (pthegner) [inline documentation/swagger.yml:2625]:
```suggestion
                      type: integer
                      minimum: 0
```

File: documentation/swagger.yml [hunk 3]
```diff
@@ -2553,6 +2554,95 @@ components:
...
+                  enum:
+                    - line
+              required:
+                - id
+                - type
+          required:
+            - start_point
+            - end_point
+            - blocked_stop_areas
+            - line
```
Comment (pthegner) [inline documentation/swagger.yml:2645]:
```suggestion
            - route_patterns
            - line
```

Comment (dvdn) [reply documentation/swagger.yml:2643]: We are waiting for 'grand chief' confirmation about mandatory or not

Comment (stifoon) [reply documentation/swagger.yml:2643]: OK : "Line" has not to be mandatory 
But could you add a comment in the documentation like :
_if "line" is filled, integration performance in navitia can be increase_

Comment (pthegner) [reply documentation/swagger.yml:2643]: @stifoon , it's more for the "route_patterns" part (required or not)

Comment (stifoon) [reply documentation/swagger.yml:2643]: "route_patterns" are required : they represent the railway which are blocked. 
* If someone want to add a "section" disruption on a bus line, he has to add a "line_section" disruption : line or route is mandatory
* If someone want to add a "section" disruption on railways, he has to add a "rail_section" disruption : route_patterns are mandatory

**Review by pthegner — APPROVED** [review]

File: documentation/swagger.yml [hunk 4]
```diff
@@ -2553,6 +2554,97 @@ components:
...
+                    - line
+              required:
+                - id
+                - type
+          required:
+            - start_point
+            - end_point
+            - blocked_stop_areas
+            - route_patterns
+            - line
```
Comment (pthegner) [inline documentation/swagger.yml:2647]:
```suggestion
          required:
            - start_point
            - end_point
            - blocked_stop_areas
            - route_patterns
```

Comment (pthegner) [reply documentation/swagger.yml:2647]: line is not required now

**Review by Fahmus — APPROVED** [review]

**Pull request merged**